front50-web/src/main/groovy/com/netflix/spinnaker/front50/controllers/V2PipelineTemplateController.java

```diff
@@ -56,8 +57,8 @@
 @Slf4j
 public class V2PipelineTemplateController {
 
-  // TODO(jacobkiefer): Does this handle all the version strings we want to support?
-  private static final String VALID_TEMPLATE_VERSION_REGEX = "^[a-zA-Z0-9-_.]+$";
+  // TODO(jacobkiefer): (PLACEHOLDER) Decide on the final set of supported tags.
+  private static final List<String> VALID_TEMPLATE_TAGS = Arrays.asList("latest", "stable", "unstable", "experimental", "test", "canary");
```

Comment:
This seems like a great list to start with.

Reply:
:+1: